app/code/community/ShipStream/Magento1/Plugin.php

```diff
@@ -356,7 +357,7 @@ public function inventoryWithLock($query)
             $this->_lockOrderImport();
             $rows = $this->call('inventory.list', empty($query['sku']) ? NULL : strval($query['sku']));
             foreach ($rows as $row) {
-                $skus[$row['sku']] = intval($row['qty_available']);
+                $skus[$row['sku']] = intval($row['qty_advertised']);
```

Comment:
$advertsied > 0 ? $advertised : -$backordered

Reply:
changes are done and pushed